src/main/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/transport/ParsedMessage.kt

```diff
@@ -90,9 +89,7 @@ class ParsedMessage private constructor(
         val type: String
 
         fun setType(type: String): T
-        fun metadataBuilder(): ImmutableMap.Builder<String, String>
-        fun addMetadataProperty(key: String, value: String): T
```

Comment:
The same here - why you decided to remove that method? It looks very helpful when we use the builder from Java  where no `apply` function exists